test/task/SingleMultisigTask.t.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.15;
 import {Test} from "forge-std/Test.sol";
 
 import {AddressRegistry as Addresses} from "src/improvements/AddressRegistry.sol";
-import {MultisigTask} from "src/improvements/task/MultisigTask.sol";
+import {MultisigTask} from "src/improvements/tasks/MultisigTask.sol";
```

Comment:
nit, so in `test` we call it `task` but in `src` we call it `tasks`? should we be consistent?

Reply:
Renamed `test/task` dir to `test/tasks` so that it's consistent: https://github.com/ethereum-optimism/superchain-ops/pull/542/commits/d1221a5b06ed13c7bdc23d3b98f3addb06fc0199